.github/config-schema.json

```diff
@@ -1022,6 +1022,28 @@
         }
       ]
     },
+    "mise": {
+      "default": {
+        "detect_extensions": [],
+        "detect_files": [
+          ".mise.toml"
```

Comment:
we should add at least `.mise.local.toml` and `.config/mise/config.toml`

Reply:
Only `.mise.local.toml` added. Detection doesn't go deeper than the current directory's contents.